### ISSUE TITLE
Push error for existing branch when git up to date - should just leave git alone.

### ISSUE THREAD

**Issue by rishson** [sidekickcode/tracker#131]
```
[cli 73622] execAtPath: git rev-parse --symbolic-full-name ''
[cli 73622] [prepush] UNEXPECTED FAILURE Error: 'Error: Command failed: /bin/sh -c git rev-parse --symbolic-full-name ''
fatal: ambiguous argument '': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
' with error code 128, in path '/Users/andy/code/gh/demo'
stdout: 

stderr: fatal: ambiguous argument '': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

    at /Users/andy/code/sidekick/sk-deployed/app/common/shell.js:40:21
    at ChildProcess.exithandler (child_process.js:723:5)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:169:7)
    at maybeClose (child_process.js:984:16)
    at Socket.<anonymous> (child_process.js:1153:11)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:166:7)
    at Pipe.close (net.js:464:12)
[cli 73622] [prepush] exiting with code '1' with msg 'null' Error: 'Error: Command failed: /bin/sh -c git rev-parse --symbolic-full-name ''
fatal: ambiguous argument '': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
' with error code 128, in path '/Users/andy/code/gh/demo'
stdout: 

stderr: fatal: ambiguous argument '': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

    at /Users/andy/code/sidekick/sk-deployed/app/common/shell.js:40:21
    at ChildProcess.exithandler (child_process.js:723:5)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:169:7)
    at maybeClose (child_process.js:984:16)
    at Socket.<anonymous> (child_process.js:1153:11)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:166:7)
    at Pipe.close (net.js:464:12)
[cli 73622] exit 1
```

**Comment by rishson** [comment]
Not passing branch name through - thought this was fixed?

**Comment by rishson** [comment]
Fixed in 0.14.5
